tendrl/node_agent/node_sync/services_and_index_sync.py

```diff
@@ -155,8 +155,28 @@ def sync(sync_ttl=None):
                         continue
                 if NS.node_context.node_id in _node_ids:
                     if sync_ttl and len(_node_ids) == 1:
-                        etcd_utils.refresh(index_key,
-                                           sync_ttl + 50)
+                        refresh_tag = True
+                        if "provisioner" in tag:
+                            _cluster = _cluster.load()
+                            if _cluster.is_managed == "yes":
+                                # Allow to claim provisioner tag only by a
+                                # node which have peer status is connected
+                                # in a managed cluster
+                                peers = NS.tendrl.objects.GlusterPeer(
+                                    node_id=NS.node_context.node_id
+                                ).load_all()
+                                for peer in peers:
+                                    if peer.hostname == 'localhost':
```

Comment:
own nodes peer hostname is localhost only

Reply:
this check only for the managed cluster, after a cluster, is managed only the node which has a peer status is "connected" can claim the provisioner.

Reply:
in unmanage cluster if any one glusterd is down anyway import will fail, so we don't need to check this